### PR TITLE
fix(components/datetime): use input event to mark picker as dirty

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/438/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/438/storybooks/ag-grid-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/438/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/438/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/438/playground/)

File: libs/components/datetime/src/lib/modules/datepicker/datepicker-input-fuzzy.directive.ts [hunk 1]
```diff
@@ -292,6 +292,10 @@ export class SkyFuzzyDatepickerInputDirective
           this.renderer.setAttribute(element, 'aria-label', value);
         });
     }
+
+    // element.addEventListener('input', () => {
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/datetime/src/lib/modules/datepicker/datepicker-input-fuzzy.directive.ts]: Should we remove this commented code?

File: libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts [hunk 2]
```diff
@@ -268,6 +268,10 @@ export class SkyDatepickerInputDirective
           this.renderer.setAttribute(element, 'aria-label', value);
         });
     }
+
+    // element.addEventListener('input', () => {
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts]: Same comment about the commented code

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]
A couple small things

File: libs/components/datetime/src/lib/modules/datepicker/datepicker-input-fuzzy.directive.ts (hunk 1 above)
Comment (Blackbaud-CoreyArcher) [reply libs/components/datetime/src/lib/modules/datepicker/datepicker-input-fuzzy.directive.ts]: removed

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

File: libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts (hunk 2 above)
Comment (Blackbaud-CoreyArcher) [reply libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts]: removed

**Pull request merged**